### ISSUE TITLE
ODT files cannot be opened from "File"

### ISSUE THREAD

**Issue by gioele** [cceh/owncloud-installation#4]
Files that can be opened from the "Documents" view cannot be opened from the "File" view. Clicking on them produces a HTTP 500 error. There is also another error present in the page that may or may not be related to this issue:

> Content Security Policy … ("script https://.... 'unsafe-eval'")
> 
> on submit attribute on DIV element

**Comment by gioele** [comment]
This problem is caused by

```
PHP Fatal error:  Class 'OCA\\Documents\\Filter_Office' not found in
    owncloud/apps/documents/ajax/download.php
```

It has been fixed in version 7.0.3 of owncloud: owncloud/documents/pull/347.

**Comment by gioele** [comment]
Fixed updating to version 7.0.4.
